hack/util-install-bundle.sh

```diff
@@ -2,14 +2,15 @@
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
 BUNDLE=$1
+NAMESPACE=${2:-build-templates}
```

Comment:
default should be current namespace, not the 'build-templates'

Reply:
👍  fixed